website/src/components/Stats/Stats.tsx

```diff
@@ -17,7 +17,19 @@ export const Stats = ({ data }: StatsProps) => {
     return null;
   }
 
-  const keys = Object.keys(data.stats_by_name);
+  const keys = Object.keys(data.stats_by_name).filter((key) => key !== "users_accepted_tos");
+
+  // Add two copies of the message_trees_states_by_lang for creating the
+  // stacked bar and the table in addition to the donough chart
+  if (keys.indexOf("message_trees_states_by_lang_table") == -1) {
```

Comment:
I think it's cleaner to add a separate component for the table rather than adding duplicate entries for the stat map.

Reply:
Sorry, but I don't understand exactly what you proposed, maybe because I'm not fluent with React components. Could you elaborate a little more?

Reply:
I mean simply add a new component for the table separately below the map for the graphs.

Reply:
Sorry, but I don't know if I understand the proposal completely. I tried to think another way of doing it, but everything I can think of would require much more code. My lack of knowledge about React is probably why I don't see a better solution. Maybe we could leave it like this and, later, you could send a pull request with a refactoring. What do you think?

Reply:
That is fine with me, thanks.